content/en/blog/_posts/2023-12-19-scheduler-queueinghint/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kubernetes v1.29: QueueingHint Brings a New Possibility to Optimize Pod Scheduling"
+date: 2023-12-19:00:00-08:00
```

Comment:
```suggestion
date: 2023-12-19T00:00-08:00
```

Reply:
@sanposhiho Would you fix the date, per [Rey's comment](https://github.com/kubernetes/website/pull/43686#discussion_r1425879411) -- the existing format is causing build failures.